build.rs

```diff
@@ -17,8 +18,13 @@ fn main() {
     }
 
     // This println!() has a special behavior, as it will set the environment
-    // variable BUILT_VERSION, so that it can be reused from the binary.
+    // variables BUILD_VERSION and BUILD_TIMESTAMP, so that it can be reused
+    // from the binary.
     // Particularly, this is used from src/main.rs to display the exact
-    // version.
-    println!("cargo:rustc-env=BUILT_VERSION={version}");
+    // version and build timestamp.
+    println!("cargo:rustc-env=BUILD_VERSION={version}");
+    println!(
+        "cargo:rustc-env=BUILD_TIMESTAMP={}",
```

Comment:
I'm afraid this will break reproducible build.

What do you need this time stamp for?

Reply:
We would like to know if there has been any modifications to the source code at build time. We could live without build_timestamp, but would it be possible to include the build hash (aka. build_version)?

Reply:
Using the commit hash is a much better indicator than a time stamp for your use case.

The hash is already there, isn't it?

Here is what I see from a build:

```
$ cloud-hypervisor --version
cloud-hypervisor v30.0-92-g38a1b45783a6
```

If the source tree has been modified (according to git), you get the `-dirty` suffix.

Reply:
> I'm afraid this will break reproducible build.

Thanks for pointing that out, I completely overlooked that possibility.